core/node/node_framework/src/implementations/layers/pk_signing_eth_client.rs

```diff
@@ -45,36 +55,89 @@ impl WiringLayer for PKSigningEthClientLayer {
     }
 
     async fn wire(self: Box<Self>, mut context: ServiceContext<'_>) -> Result<(), WiringError> {
-        let private_key = self.wallets.operator.private_key();
-        let gas_adjuster_config = self
-            .eth_sender_config
-            .gas_adjuster
-            .as_ref()
-            .context("gas_adjuster config is missing")?;
-        let EthInterfaceResource(query_client) = context.get_resource().await?;
+        let _signing_client = match self.as_ref().client_type {
```

Comment:
Why do we need to assign this variable here?

Reply:
good catch! removed!